tools/k8s-test/daemonset.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"context"
+
+	v1beta1 "k8s.io/api/extensions/v1beta1"
+)
+
+func GetDaemonSet(ctx context.Context, name string) (*v1beta1.DaemonSet, error) {
+	obj := new(v1beta1.DaemonSet)
+	ok, err := getObject(ctx, "ds", name, obj, nil)
+	if err != nil {
+		return nil, err
+	}
+	if !ok {
+		return nil, NotFound.New("no such daemon set %q", name)
+	}
+	return obj, nil
+}
+
+func GetDaemonSets(ctx context.Context) ([]v1beta1.DaemonSet, error) {
```

Comment:
is it used? or it was used for future uses?

Reply:
Not used yet. I implemented a bunch of functions when going a certain direction. When I shifted direction I didn't bother cleaning them out. I'll remove it. It is trivial to add back in.